nix/desktop/linux/linuxdeployqt/default.nix

```diff
@@ -3,17 +3,18 @@
 with pkgs;
 
 stdenv.mkDerivation rec {
```

Comment:
Why `rec` instead of `let`?

Reply:
why would a `let` be better in this case?

Reply:
Not sure, I just thought it was the preferred way of preparing variables for use before the actual call. I have no strong opinion on this, which is why my comment ends with a question mark =].

Reply:
This is exactly why the `rec` keyword exists, so that you can reuse fields from the attribute set recursively without needing to define a `let` section.

Reply:
Fair enough, good PR!